src/features/cookies/helpers/startTrackingAcceptedCookies.test.ts

```diff
@@ -57,4 +61,10 @@ describe('startTrackingAcceptedCookies', () => {
     expect(amplitude.disableCollection).toHaveBeenCalled()
     expect(analytics.disableCollection).toHaveBeenCalled()
   })
+
+  it('should remove generate algolia key from localStorage if refused algolia tracking (performance)', () => {
+    startTrackingAcceptedCookies(COOKIES_BY_CATEGORY.marketing) // without performance cookies and algolia insight
```

Comment:
```suggestion
    const performanceRefused = [] // ou performanceNotAccepted

    startTrackingAcceptedCookies(performanceRefused)
```

Reply:
Ça revient a tester le premier it 
```
  it('should disable tracking if refused all cookies', () => {
    startTrackingAcceptedCookies([])

    expect(amplitude.disableCollection).toHaveBeenCalled()
    expect(analytics.disableCollection).toHaveBeenCalled()
    expect(campaignTracker.startAppsFlyer).toHaveBeenCalledWith(false)
    expect(Batch.optOut).toHaveBeenCalled()
  })
```
Mais il manquerait un test lorsque l'utilisateur fait des choix custom